nc-news/components/Article.jsx

```diff
@@ -3,33 +3,48 @@ import axios from "axios";
 import { Routes, Route, useParams } from "react-router-dom";
 import Button from "react-bootstrap/Button";
 import Card from "react-bootstrap/Card";
+import api from "../utils/api";
 
 const Article = () => {
   const [article, setArticle] = useState({});
   const [comments, setComments] = useState([]);
+  const [articleVotes, setArticleVotes] = useState(0);
 
   let { article_id } = useParams();
 
   useEffect(() => {
+    api.getArticle(article_id).then((data) => {
+      setArticle(data.article);
+      setArticleVotes(data.article.votes);
+    });
+    api.getComments(article_id).then((comments) => {
+      setComments(comments);
+    });
+  }, [article_id]);
+
+  const handleUpvote = () => {
+    setArticleVotes((currentVotes) => currentVotes + 1);
     axios
-      .get(
-        `https://nc-news-server-utuo.onrender.com/api/articles/${article_id}`
+      .patch(
+        `https://nc-news-server-utuo.onrender.com/api/articles/${article_id}`,
+        { inc_votes: 1 }
       )
-      .then((response) => {
-        setArticle(response.data.article);
+      .catch((error) => {
+        console.log(error);
       });
-  }, []);
+  };
 
-  useEffect(() => {
+  const handleDownvote =()=>{
+    setArticleVotes((currentVotes) => currentVotes - 1);
     axios
-      .get(
-        `https://nc-news-server-utuo.onrender.com/api/articles/${article_id}/comments`
+      .patch(
+        `https://nc-news-server-utuo.onrender.com/api/articles/${article_id}`,
+        { inc_votes: -1 }
       )
-      .then((response) => {
-        setComments(response.data.comments);
-        console.log(comments);
-      });
-  }, []);
+      .catch((error) => {
+        console.log(error);
+      })
+  }
```

Comment:
You could combine both vote actions into one function, if you accept a parameter representing either 1 or -1, and pass that through in the onClick, you could use that value instead of hardcoding 1 or -1 in each these vote functions which end up looking very similar